Documentation/concepts/networking/masquerading.rst

```diff
@@ -114,7 +114,23 @@ In addition, if the ``masqLinkLocal`` is not set or set to false, then
 The agent uses Fsnotify to track updates to the configuration file, so the original
 ``resyncInterval`` option is unnecessary.
 
-The example below shows how to configure the agent via :term:`ConfigMap` and to verify it:
+The example below shows how to configure the agent When you install cilium via :term:`helm install` and to verify it:
+
+.. code-block:: shell-session
```

Comment:
I would add this section after the installation via rfc1918.yaml, and then phrase it like:

> Alternatively, you can pass `--set ipMasqAgent.config.nonMasqueradeCIDRs='{172.16.0.0/12,192.168.0.0/16}'` and  `--set ipMasqAgent.config.masqLinkLocal=false` when installing Cilium via Helm to configure the ip-masq-agent.

Reply:
Great! Thanks @brb